.travis.yml

```diff
@@ -1,4 +1,4 @@
-language: python
+		language: python
```

Comment:
Not sure what this tab is for and if this is an issue.

Reply:
fixed